Week4/README.md

```diff
@@ -4,27 +4,22 @@
 
 These are the topics for week 4:
 
-1. [Higher Order Functions](https://study.hackyourfuture.net/#/javascript/higher-order-functions)
-   - Array methods
-2. [Closures](https://study.hackyourfuture.net/#/javascript/closures)
-   - Execution context
-   - Why do we need closures?
-3. [Problem solving](https://study.hackyourfuture.net/#/programming/problem-solving-process)
-4. [Unit testing](https://study.hackyourfuture.net/#/testing/unit-tests)
-   - [Testing framework (Jest)](https://study.hackyourfuture.net/#/tools/test-framework)
-5. [Finalise Your personal brand](https://github.com/HackYourFuture/yourpersonalbrand)
+1. [The `this` keyword](https://study.hackyourfuture.net/#/javascript/this)
+1. [Constructor functions](https://study.hackyourfuture.net/#/javascript/constructor-functions)
```

Comment:
For the 'Miss Xing' video in the study book you may want to suggest students to turn on subtitles in YouTube if they have difficulty understanding her foreign accent.